api/src/main/java/org/openmrs/module/reporting/cohort/definition/library/BuiltInCohortDefinitionLibrary.java

```diff
@@ -105,6 +107,16 @@ public CohortDefinition getAnyEncounterDuringPeriod() {
         return new MappedParametersCohortDefinition(cd, "onOrAfter", "startDate", "onOrBefore", "endDate");
     }
 
+    @DocumentedDefinition("codedObsSearchAdvanced")
+    public CohortDefinition getPatientsHavingObs() {
+        CodedObsCohortDefinition cd = new CodedObsCohortDefinition();
```

Comment:
The method name in a library generally should match the documented definition key.  I'm not quite sure what is advanced about this, or what is "coded" about it, since there nothing that allows one to use this to query for particular coded answer values.  Is the idea that we'll keep adding new parameters to this existing definition as needed?  @djazayeri